WordPressFlux/Package.swift

```diff
@@ -5,6 +5,7 @@ import PackageDescription
 
 let package = Package(
     name: "WordPressFlux",
+    platforms: [.iOS(.v13)],
```

Comment:
This was necessary to avoid a warning that would become an error with the new settings. It's also good practice, even though we don't need to expose the Package to the outside world. 

Fun fact, without this specified, the default value (as reported in the error) was iOS 4!